flow/project.py

```diff
@@ -907,7 +914,9 @@ def _fetch_scheduler_status(self, jobs=None, file=None, ignore_errors=False):
             scheduler_info = {sjob.name(): sjob.status() for sjob in self.scheduler_jobs(scheduler)}
             status = dict()
             print(self._tr("Query scheduler..."), file=file)
-            for op in self.next_operations(*jobs):
+            for op in tqdm(self.next_operations(*jobs),
```

Comment:
Is the additional progress bar insightful for the user or did you primarily use it during debugging?

Reply:
I created it for debugging, but I actually do think that it's helpful. However, the utility is dependent on the data space being quite large; for <1000 jobs this step is typically instant. I left it in intentionally, but if you are concerned that it's too verbose have a look at the sample output and we can make a call.

Reply:
We can leave it in if you think that it is helpful not a distraction.